.github/workflows/build-crowdin.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Crowdin project
+
+on:
+  schedule:
+    - cron: "20 04 1 * *"
+  workflow_dispatch:
```

Comment:
Was reading a bit about this, pretty cool. This would allow us to trigger the workflow manually if necessary, right?